src/77480.c

```diff
@@ -1,14 +1,19 @@
 #include "common.h"
 #include "ld_addrs.h"
 
-extern s32 D_8010C920;
+#define E21870_VRAM_DEF 0x802B7000
+#define E225B0_VRAM_DEF 0x802B7000
```

Comment:
It doesn't match to use ld_addrs.h, right?

Reply:
yeah, it doesn't - talked to Josh about this earlier